### DISCUSSION TITLE
如何关闭阿里云盘预加载存储

### DISCUSSION THREAD

**Discussion by Cxyofficial** [AlistGo/alist#6421] [General]
每次在主页刷新一次就会把阿里云盘的api访问一遍 访问次数多了就会failed to refresh token: Too Many Requests
能不能改成点击进文件夹后再加载阿里云盘api 而不是刷新一次网页加载一次

**Comment by pongfcnkl** [comment]
缓存不改0？

**Reply by Cxyofficial** [reply]
改成0就可以解决了吗
